install/runtime-payload/hack/binaries.sh

```diff
@@ -8,15 +8,17 @@ script_dir=$(dirname "$(readlink -f "$0")")
 bin_dir=${script_dir}/../bin
 kata_dir=${script_dir}/../../../../kata-containers
 
-function build_kata_runtime() {
-	pushd "${kata_dir}/src/runtime"
-        make
-	cp containerd-shim-kata-v2 ${bin_dir}/ 
+function build_kata_shim_v2() {
+	pushd "${kata_dir}/src/runtime/cmd/containerd-shim-kata-v2"
+		env GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o containerd-shim-kata-v2-x86_64
```

Comment:
minor: Perhaps, it might be worth to use a `for` loop over the `amd64` and `s390x` to not duplicate the same commands for each architecture and to easily enable other archs in future? Wdyt?

Reply:
updated.